manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -84,6 +84,13 @@ spec:
             runAsGroup: 0
             runAsUser: 0
             runAsNonRoot: false
+            capabilities:
+              drop:
+              - ALL
+              add:
+              - NET_ADMIN
+              - SYS_ADMIN
+              - NET_RAW
```

Comment:
I want to do that in a followup, and drop the `privileged` flag, as mentioned.

Sidecar repair also requires (some but not all) of these, and I would rather use this mechanism (DROP_ALL then use templating to add back what explicit capabilities are needed for ambient, repair, both, or neither)

Reply:
I would prefer to discuss PRs that impact all production users in a small targetted PR instead of in this massive PR so we can give it proper attention (and not risk delaying this PR)

Reply:
https://github.com/istio/istio/issues/49004

raised a ticket - ty!